app/src/main/java/com/github/h3lp3rs/h3lp/forum/CachedForum.kt

```diff
@@ -33,9 +33,9 @@ class CachedForum(private val forum: Forum) : Forum {
     data class CacheHeader(val categoryPaths: ArrayList<String>)
 
     @RequiresApi(Build.VERSION_CODES.O)
-    override fun newPost(author: String, content: String): CompletableFuture<ForumPost> {
+    override fun newPost(author: String, content: String, isPost: Boolean): CompletableFuture<ForumPost> {

```

Comment:
I would just add a more relevant function name. 
Indeed newPost and isPost is a bit contradictory!
But don't change it, now we're used to it.

Reply:
Also, this file gets deleted by me (PR), to increase the coverage a bit. I'm keeping it under my sleeve for next sprint (integration)